common/src/main/java/com/netflix/conductor/common/utils/TaskUtils.java

```diff
@@ -28,4 +40,15 @@ public static String removeIterationFromTaskRefName(String referenceTaskName) {
         String[] tokens = referenceTaskName.split(TaskUtils.LOOP_TASK_DELIMITER);
         return tokens.length > 0 ? tokens[0] : referenceTaskName;
     }
+
+    /**
+     * This boolean decides whether to index tasks on communication failure or to index always.
+     * Default value is set to true in the config file. If the value is true, indexing will be done
+     * only on communication failure. It its false, all tasks will be indexed always.
+     *
+     * @return
+     */
+    public static boolean indexOnCommunicationFailure() {
```

Comment:
I think method can be named as "isIndexOnCommunicationFailure()".

Reply:
done